internal/db/models/instance.go

```diff
@@ -38,16 +40,70 @@ const (
 // Instance represents a compute instance in the system
 type Instance struct {
 	gorm.Model
-	JobID      uint           `json:"job_id" gorm:"not null;index"` // ID from the jobs table
-	ProviderID ProviderID     `json:"provider_id" gorm:"not null"`
-	Name       string         `json:"name" gorm:"not null;index"`
-	PublicIP   string         `json:"public_ip" gorm:"varchar(100)"`
-	Region     string         `json:"region" gorm:"varchar(255)"`
-	Size       string         `json:"size" gorm:"varchar(255)"`
-	Image      string         `json:"image" gorm:"varchar(255)"`
-	Tags       pq.StringArray `json:"tags" gorm:"type:text[]"`
-	Status     InstanceStatus `json:"status" gorm:"index"`
-	CreatedAt  time.Time      `json:"created_at" gorm:"index"`
+	JobID         uint           `json:"job_id" gorm:"not null;index"`
+	ProviderID    string         `json:"provider_id" gorm:"not null"`
+	Name          string         `json:"name" gorm:"not null;index"`
+	PublicIP      string         `json:"public_ip" gorm:"varchar(100)"`
+	Region        string         `json:"region" gorm:"varchar(255)"`
+	Size          string         `json:"size" gorm:"varchar(255)"`
+	Image         string         `json:"image" gorm:"varchar(255)"`
+	Tags          pq.StringArray `json:"tags" gorm:"type:text[]"`
+	Status        InstanceStatus `json:"status" gorm:"index"`
+	Volumes       pq.StringArray `json:"volumes" gorm:"type:text[]"`
+	VolumeDetails VolumeDetails  `json:"volume_details" gorm:"type:jsonb"`
+	CreatedAt     time.Time      `json:"created_at" gorm:"index"`
+}
+
+// VolumeDetail represents the details of a volume attached to an instance
+type VolumeDetail struct {
+	ID         string `json:"id"`          // Volume ID
+	Name       string `json:"name"`        // Volume name
+	SizeGB     int    `json:"size_gb"`     // Size in gigabytes
+	Region     string `json:"region"`      // Region where the volume is created
```

Comment:
How can a volume be in a different region than the instance? 
I don't think this is required.